### PR TITLE
bump to 20180501

### PR DESCRIPTION
**Pull request by carlodri**

### CODE REVIEW

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 package:
   name: x264
-  version: 20131217
+  version: {{ version }}
 
 source:
   fn: x264-snapshot-{{ version }}-2245-stable.tar.bz2
   url: http://download.videolan.org/pub/videolan/x264/snapshots/x264-snapshot-{{ version }}-2245-stable.tar.bz2
-  md5: cfd27222c7aa9983259ddb10f7f570a6
+  md5: 89fb34fe9d0959894592e308cafff5ad
```
Comment (jakirkham) [inline recipe/meta.yaml]: Could we switch to a more secure checksum?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 
 build:
-  number: 3
+  number: 0
   skip: true         # [win]
 
 requirements:
   build:
     - toolchain
     - perl 5.20.3.1
```
Comment (jakirkham) [inline recipe/meta.yaml]: With the new `conda-smithy`, we can drop this hacky `perl` pinning.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
-    - yasm
+    - nasm
```
Comment (jakirkham) [inline recipe/meta.yaml]: Note to others: Upstream switched from `yasm` to `nasm`. ( https://stackoverflow.com/a/44573541 )

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
+  md5: 89fb34fe9d0959894592e308cafff5ad
 
 build:
-  number: 3
+  number: 0
   skip: true         # [win]
 
 requirements:
   build:
     - toolchain
```
Comment (jakirkham) [inline recipe/meta.yaml]: Also we can change this to `{{ compiler("c") }}` and then re-render. This is part of a compiler transition that we are making throughout conda-forge.

**Comment by jakirkham** [comment]
Thanks @carlodri.

Could you please resolve merge conflicts with `master`? Made a few other comments above. Hopefully nothing too complicated.

Any interest in helping out with maintenance of this feedstock?

**Comment by carlodri** [comment]
@jakirkham thanks for the feedback! I've hopefully fixed merge conflicts and added myself as maintainer

**Comment by carlodri** [comment]
Oh sorry, still missed the compiler thing, will do tomorrow...

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
Thanks @carlodri. Looks good.

Went ahead and added the `compiler` change to your PR. Bot to re-render. Hope that is ok.

**Comment by jakirkham** [comment]
Would like to sort out how we pin `x264` in `ffmpeg` before merging this. Have raised issue ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/51 ) to discuss how best to pin `x264`.

**Comment by carlodri** [comment]
@jakirkham can this be merged? I think `ffmpeg` shouldn't have problems with the new version, but we can try and if things go wrong we pin accordingly.

**Comment by jakirkham** [comment]
So I'm 99% certain this will break `ffmpeg` as the SONAME has changed a dozen times (amongst other breaking changes) since the release in here and the current one. I'd rather us start by determining how we want to pin things ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/51 ) and apply that pinning where needed before updating. It may still break things unfortunately, but we will be down to pulling unpinned `ffmpeg` packages at that point, which is something we will need to do anyways.

ref: https://abi-laboratory.pro/tracker/timeline/x264/

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

**Comment by carlodri** [comment]
supersedes and includes changes from #15

**Pull request merged**